### PR TITLE
fix: do not validate mapsheets against invalid ranges

### PR DESCRIPTION
**Pull request by blacha**
#### Motivation

the validation logic for mapsheets is currently giving incorrect results, this disables the validation logic until a better fix can be implemented.

#### Modification

removes the invalid magic numbers from mapsheet validation.

#### Checklist

_If not applicable, provide explanation of why._

- [ ] Tests updated
- [ ] Docs updated
- [ ] Issue linked in Title

### CODE REVIEW

File: src/commands/tileindex-validate/__test__/tileindex.validate.test.ts [hunk 1]
```diff
@@ -49,14 +49,31 @@ describe('getTileName', () => {
     assert.equal(convertTileName('AT24_50000_0101', 50000), 'AT24');
     assert.equal(convertTileName('AT25_50000_0101', 50000), 'AT25');
     assert.equal(convertTileName('CK08_50000_0101', 50000), 'CK08');
+    assert.equal(convertTileName('CK08_50000_0101', 50000), 'CK08');
```
Comment (l0b0) [inline src/commands/tileindex-validate/__test__/tileindex.validate.test.ts:52]: This is identical to the line above. Is that intentional?

Comment (blacha) [reply src/commands/tileindex-validate/__test__/tileindex.validate.test.ts:52]: nope.

File: src/commands/tileindex-validate/__test__/tileindex.validate.test.ts [hunk 2]
```diff
@@ -49,14 +49,31 @@ describe('getTileName', () => {
...
       assert.equal(getTileName(sheet.origin.x, sheet.origin.y, 5000), sheet.code + '_5000_0101');
       assert.equal(getTileName(sheet.origin.x, sheet.origin.y, 1000), sheet.code + '_1000_0101');
+      assert.equal(getTileName(sheet.origin.x, sheet.origin.y, 500), sheet.code + '_500_001001');
+    });
+
+    it('should get the bottom right 1:50k, 1:10k, 1:5k, 1:1k for ' + sheet.code, () => {
+      // for each scale calculate the bottom right tile then find the mid point of it
+      // then look up the tile name from the midpoint and ensure it is the same
+      for (const scale of [10_000, 5_000, 1_000, 500] as const) {
+        const tileCount = 50_000 / scale;
```
Comment (l0b0) [inline src/commands/tileindex-validate/__test__/tileindex.validate.test.ts:69]: We could reuse `mapSheetTileGridSize` here.

**Comment by blacha** [comment]
going to replace this PR with https://github.com/linz/argo-tasks/pull/863

**Comment by blacha** [comment]
whoops #864

**Pull request closed without merging**